doc/tutorials/nires_howto.rst

```diff
@@ -0,0 +1,435 @@
+
+.. include:: ../include/links.rst
+
+.. _nires_howto:
+
+================
+Keck/NIRES HOWTO
+================
+
+Overview
+========
+
+This doc goes through a full run of PypeIt on one of the Keck/NIRES datasets
+(specifically the ``NIRES`` dataset) in the `PypeIt Development Suite`_ (see
+:ref:`here <dev-suite>`).
+
+----
+
+Setup
+=====
+
+Before reducing your data with PypeIt, you first have to prepare a
+:ref:`pypeit_file` by running :ref:`pypeit_setup`:
+
+.. code-block:: bash
+
+    pypeit_setup -r $HOME/Work/packages/PypeIt-development-suite/RAW_DATA/keck_nires/NIRES/ -s keck_nires -b -c A 
+
+where the ``-r`` argument should be replaced by your local directory and the
+``-b`` indicates that the data uses background images and should include the
+``calib``, ``comb_id``, ``bkg_id`` in the pypeit file.  This directory only has
+one instrument configuration (NIRES only has one configuration anyway), so
+setting ``-c A`` and ``-c all`` is equivalent.
+
+This will make a directory called ``keck_nires_A`` that holds a pypeit file
+called ``keck_nires_A.pypeit`` that looks like this:
+
+.. include:: ../include/keck_nires_A.pypeit.rst
+
+For this example dataset, the details of the default pypeit file are not
+correct; see :ref:`here <nires_config_report>` for an example where the frame
+typing and dither pattern determinations *are* correct by default.
+
+The issue with this dataset is that only two pointings in the ABBA pattern for
```

Comment:
We should just do automated frame typing based on exposure time. It really depends on how long these standard star exposures are.

Reply:
Exactly.  The current default is that any standard exposures are <60s long.  The ones in this example are 200s, which is why they're typed as `science`.  So either this NIRES dataset is atypical, or we should change the default exposure time used to identify standards.  Do you have a sense of what this number should be?